From 31ec05143136701d6797a5d92d9e94b9cd4ed438 Mon Sep 17 00:00:00 2001
From: Robert Wen <robertwen20@gmail.com>
Date: Mon, 1 Dec 2014 15:44:43 -0500
Subject: [PATCH 04/16] Added a FrozenState to MinionState
The creation of the FrozenState class will implement the MinionState interface, by doing that a Minion can have frozen as a state. 
---
 src/main/java/com/hearthsim/card/minion/FrozenState.java | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 src/main/java/com/hearthsim/card/minion/FrozenState.java

diff --git a/src/main/java/com/hearthsim/card/minion/FrozenState.java b/src/main/java/com/hearthsim/card/minion/FrozenState.java
new file mode 100644
index 0000000..dcbc585
--- /dev/null
+++ b/src/main/java/com/hearthsim/card/minion/FrozenState.java
@@ -0,0 +1,10 @@
+package com.hearthsim.card.minion;
+
+public class FrozenState implements MinionState{
+	
+	public Minion handleStateEffect(Minion m) {
+		m.hasAttacked_ = true;
+		return m;
+	}
+
+}
-- 
1.8.3.msysgit.0

